ui-new/start.html

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
This one is not related to this start.html but to the one that lives inside the html folder. Your commits should be removing the html from that to their own separate file but for some reason they are still present there.